packages/antd/test/__snapshots__/Form.test.js.snap

```diff
@@ -522,37 +522,23 @@ exports[`single fields format color 1`] = `
   <div
     className="form-group field field-string"
   >
-    <label
-      className="ant-checkbox-wrapper"
+    <input
```

Comment:
I don't really understand why this changed. Seems like it might be a problem that we lost the `label` tag?

Reply:
I think it has something to do with the old `Color` widget importing a different input, which in my mind seems wrong.
```
import Input from "antd/lib/checkbox";
```

I'll test it in the old and new playground and see...

Reply:
The old playground just looks weird with a checkbox and plain-old doesn't actually show the selected color at all:
<img width="261" alt="Screen Shot 2022-08-07 at 12 58 35 AM" src="https://user-images.githubusercontent.com/51679588/183281628-39849089-a0eb-4fcc-b71f-4f300451f6a5.png">

Where as this implementation does:
<img width="744" alt="Screen Shot 2022-08-07 at 1 12 19 AM" src="https://user-images.githubusercontent.com/51679588/183281752-eed7a486-0ecc-4a55-a271-ff361a923156.png">

Reply:
Still looks weird, but better I guess, so let's go with it